### ISSUE TITLE
References to MS-SQL built-in schema create warnings?

### ISSUE THREAD

**Issue by kevin-david** [rr-wfm/MSBuild.Sdk.SqlProj#101]
Great project! Unblocked my ability to build SSDT projects on my Mac as well as Windows 👍 

Not sure there's an easy way to fix this, but figured it was worth reporting. I have some references to built-in SQL constructs and they create warnings during the build. Here's what they look like:

```
prc_DeleteProgram.sql(26,9): ModelValidationError warning SQL71502: SqlProcedure: [dbo].[prc_DeleteProgram] contains an unresolved reference to an object. Either the object does not exist or the reference is ambiguous because it could refer to any of the following objects: [INFORMATION_SCHEMA].[COLUMNS].[progCol]::[TABLE_SCHEMA] or [INFORMATION_SCHEMA].[COLUMNS].[TABLE_SCHEMA]. 

prc_DeleteProgram.sql(47,7): ModelValidationError warning SQL71502: SqlProcedure: [dbo].[prc_DeleteProgram] has an unresolved reference to object [dbo].[sp_executesql].

prc_DeleteProgram.sql(35,7): ModelValidationError warning SQL71502: SqlProcedure: [dbo].[prc_DeleteProgram] contains an unresolved reference to an object. Either the object does not exist or the reference is ambiguous because it could refer to any of the following objects: [INFORMATION_SCHEMA].[COLUMNS].[progCol]::[TABLE_CATALOG] or [INFORMATION_SCHEMA].[COLUMNS].[TABLE_CATALOG]
```

These are caused by doing something like this:
```sql
DECLARE table_cursor CURSOR FOR
SELECT  progCol.TABLE_SCHEMA + '.' + progCol.TABLE_NAME
FROM    INFORMATION_SCHEMA.COLUMNS progCol
JOIN    INFORMATION_SCHEMA.COLUMNS tenantCol
ON      tenantCol.COLUMN_NAME = 'TenantId'
	AND progCol.TABLE_CATALOG = DB_NAME()
        AND progCol.DATA_TYPE = 'int'
        AND tenantCol.TABLE_NAME = progCol.TABLE_NAME
WHERE   progCol.COLUMN_NAME = 'ProgramId'
	AND progCol.TABLE_NAME <> 'tbl_Program'
	AND progCol.TABLE_CATALOG = DB_NAME()
	AND progCol.TABLE_NAME LIKE 'tbl_%'
        AND progCol.DATA_TYPE = 'int'
```

Is this something I'm doing wrong, or perhaps a limitation of the current project SDK?

Thanks!

**Comment by kevin-david** [comment]
Just realized this is a duplicate of https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/issues/64 - sorry :)
